From: Felipe Balbi <balbi@ti.com>
Date: Thu, 17 Dec 2015 09:55:41 -0600
Subject: usb: of: fix build breakage on !OF
Git-commit: be99c84300950e876074916b215b511f69f83d3b
Patch-mainline: v4.5-rc1
References: bsc#1033254

If OF is disabled, we will try to define a stub for
of_usb_get_dr_mode_by_phy(), however that missed a
static inline annotation which made us redefine the
stub over and over again. Fix that.

Fixes: 98bfb3946695 ("usb: of: add an api to get
	dr_mode by the phy node")
Signed-off-by: Felipe Balbi <balbi@ti.com>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 include/linux/usb/of.h |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/include/linux/usb/of.h
+++ b/include/linux/usb/of.h
@@ -19,7 +19,8 @@ int of_usb_update_otg_caps(struct device
 struct device_node *usb_of_get_child_node(struct device_node *parent,
 			int portnum);
 #else
-enum usb_dr_mode of_usb_get_dr_mode_by_phy(struct device_node *np, int arg0)
+static inline enum usb_dr_mode
+of_usb_get_dr_mode_by_phy(struct device_node *np, int arg0)
 {
 	return USB_DR_MODE_UNKNOWN;
 }
